pages/linux/logname.md

```diff
@@ -0,0 +1,7 @@
+# logname 
+
+> Shows the users login name.
```

Comment:
Perhaps `user's` instead of `users` would read better?

Reply:
Yup. Done that.